features/page_objects/journey/standard_page.rb

```diff
@@ -10,6 +10,7 @@ class StandardPage < SitePrism::Page
   element(:content, "#content")
   element(:button, ".button")
   element(:submit_button, "input[type='submit']")
+  element(:registration_number, "#reg_identifier")
```

Comment:
How do we know that this will be a standard id? Which pages will it appear on?

Reply:
All confirmation pages. Shall I create a `StandardConfirmationPage` object for it? Would probably make more sense?

Reply:
Good idea although I'd prefer to just call it `ConfirmationPage` . We have other 'standard' pages like AddressLookupPage which are reused, so I'm happy to exclude Standard here.

Reply:
Cool :) Will do that